internal/events/eventstest/events_test_utils.go

```diff
@@ -114,6 +114,19 @@ func WithMessageContainsMatcher(expected string) eventPartMatcher {
 	}
 }
 
+func WithInfoMatcher(expected string) eventPartMatcher {
+	return func(event interface{}) bool {
+		e, ok := event.(eventsapi.HostUpdateStatusEvent)
```

Comment:
If we change the interface to only ensure we have a `GetInfo` method then this becomes much less specific to the event type.

Reply:
@nmagnezi Nick has been helping me with my PR and we'd like to get your input on these thoughts if OK?

Reply:
I see those specific tests, as opposed to other tests, don't really use the db. so yes - `GetInfo` makes sense and it should be enough to test against it.